pkg/vsphere/extraconfig/keys.go

```diff
@@ -27,30 +27,73 @@ import (
 )
 
 const (
-	// DefaultTagName value
+	// GuestInfoPrefix is dictated by vSphere
+	GuestInfoPrefix = "guestinfo."
+
+	// ScopeTag is the tag name used for declaring scopes for a field
+	ScopeTag = "scope"
+	// KeyTag is the tag name by which to override default key naming based on field name
+	KeyTag = "key"
+	// RecurseTag is the tag name with which different recursion properties are declared
+	RecurseTag = "recurse"
+
+	// HiddenScope means the key is hidden from the guest and will not have a GuestInfoPrefix
+	HiddenScope = "hidden"
+	// ReadOnlyScope means the key is read-only from the guest
+	ReadOnlyScope = "read-only"
```

Comment:
As above, `keys-test.go` contains hard-coded instances of this string (and others, below).